backend/src/services/v2/review.ts

```diff
@@ -132,9 +139,38 @@ export async function respondToReview(
   if (!update) {
     throw GenericError(500, `Adding response to Review was not successful`, { modelId, reviewIdQuery, role })
   }
+  sendReviewNotification(update, user)
   return update
 }
 
+export async function sendReviewNotification(review: ReviewDoc, user: UserDoc) {
+  let reviewIdQuery
+  switch (review.kind) {
+    case ReviewKind.Access: {
+      if (!review.accessRequestId) {
+        log.info('Cannot find modelId')
```

Comment:
I'd be tempted to upgrade this log to an error. We've only added this if statement to be type safe and it shouldn't ever happen. Also maybe change the log message to 'Unable to send notification for review response. Cannot find access request ID.